**1. State the Hypotheses statement:**

**Null Hypothesis(H0):**

*The actual mean weekly operating cost follows cost model: W=1000+5X*

*HO: μ=1000+5X*

**Alternative Hypothesis (H₁):**

*The actual mean weekly operating cost is greater than predicted cost model*

*H1: μ>1000+5X*

*This is a one-tailed test since we are testing whether the actual cost is higher.*




**2.Calculate the Test Statistics:**
Sample mean(𝑥)  = Rs. 3,050

Theoretical mean weekly cost (μ0)=1000+5X , X=600

μ0=1000+5(600)=Rs.4000

Sample size n=25

Population standard deviation (𝜎) = 5×25=125

Test static=(x-μ0)/(𝜎/sq.rt(n))

=(3050-4000)/(125/5)

=-950/25

***Test Statistic (z-value): -38.0***
​

In [3]:
# Calcualting with Python
import math

# Given data
sample_mean_cost = 3050  # Sample mean weekly cost (Rs)
theoretical_units_produced = 600  # Mean units produced in a week (μ)
std_dev_units = 25  # Standard deviation of units produced (σ)
sample_size = 25  # Sample size (n)

# Theoretical weekly operating cost model: W = $1,000 + $5X
theoretical_mean_cost = 1000 + 5 * theoretical_units_produced  # Theoretical weekly cost for X = 600 units

# Standard deviation of the weekly cost (σ_w = 5 * σ)
std_dev_weekly_cost = 5 * std_dev_units

# Calculate the test statistic (t)
t_statistic = (sample_mean_cost - theoretical_mean_cost) / (std_dev_weekly_cost / math.sqrt(sample_size))

print(f"Test statistic: {t_statistic}")


Test statistic: -38.0


**3.Determine the Critical Value:**

*Alpha (α) Level:*

*The problem specifies an alpha level of 5% (α = 0.05), which represents the probability of rejecting the null hypothesis when it is actually true (Type I error).
This is a one-tailed test, as we are only interested in whether the weekly operating costs are higher than the model suggests.*

*Critical Value for a One-Tailed Test:*

*For a one-tailed test with α = 0.05, we need to find the value from the standard normal (Z) distribution that leaves 5% of the probability in the right tail.
Using the Z-distribution table or the statistical function ppf (percent point function), we calculate the critical value.*

*Formula to Calculate Critical Value: The formula for calculating the critical value from the standard normal distribution is:*

Zα=ppf(1−α)

In [1]:
from scipy import stats
# Given alpha level for one-tailed test
alpha = 0.05
# Calculate the critical value for a one-tailed test using the ppf (percent point function)
critical_value = stats.norm.ppf(1 - alpha)

print(f"Critical value: {critical_value}")


Critical value: 1.6448536269514722


**Decision**
*The critical value for α = 0.05 is 1.645. This means that if our test statistic is greater than 1.645, we would reject the null hypothesis.*

**Conclusion:** *Based on this analysis, there is no strong evidence to support the restaurant owners' claim that the weekly operating costs are higher than the model suggests. The observed difference is not statistically significant.*